locales/pl.json

```diff
@@ -1,155 +1,155 @@
+	"Editor Keymap": "Układ klawiszy edytora",
+	"default": "domyślny",
 	"vim": "vim",
 	"emacs": "emacs",
-	"⚠️ Please restart boostnote after you change the keymap": "⚠️ Please restart boostnote after you change the keymap",
-	"Show line numbers in the editor": "Show line numbers in the editor",
-	"Allow editor to scroll past the last line": "Allow editor to scroll past the last line",
-	"Bring in web page title when pasting URL on editor": "Bring in web page title when pasting URL on editor",
-	"Preview": "Preview",
-	"Preview Font Size": "Preview Font Size",
-	"Preview Font Family": "Preview Font Family",
-	"Code block Theme": "Code block Theme",
-	"Allow preview to scroll past the last line": "Allow preview to scroll past the last line",
-	"Show line numbers for preview code blocks": "Show line numbers for preview code blocks",
-	"LaTeX Inline Open Delimiter": "LaTeX Inline Open Delimiter",
-	"LaTeX Inline Close Delimiter": "LaTeX Inline Close Delimiter",
-	"LaTeX Block Open Delimiter": "LaTeX Block Open Delimiter",
-	"LaTeX Block Close Delimiter": "LaTeX Block Close Delimiter",
-	"PlantUML Server": "PlantUML Server",
-	"Community": "Community",
-	"Subscribe to Newsletter": "Subscribe to Newsletter",
+	"⚠️ Please restart boostnote after you change the keymap": "⚠️ Wymagane zresetowania Boostnote po zmianie ustawień klawiszy",
+	"Show line numbers in the editor": "Pokazuj numery lini w edytorze",
+	"Allow editor to scroll past the last line": "Pozwalaj edytorowi na przewijanie do końca lini",
+	"Bring in web page title when pasting URL on editor": "Wprowadź tytuł wklejanej strony WWW do edytora",
+	"Preview": "Podgląd",
+	"Preview Font Size": "Podgląd rozmiaru czcionki",
+	"Preview Font Family": "Podgląd czcionki",
+	"Code block Theme": "Wygląd bloku kodu",
+	"Allow preview to scroll past the last line": "Pozwalaj poglądowi na przewijanie do końca lini",
+	"Show line numbers for preview code blocks": "Pokazuj numery lini dla podglądu bloków kodu",
+	"LaTeX Inline Open Delimiter": "Otwarcie liniowego ogranicznika LaTeX",
+	"LaTeX Inline Close Delimiter": "Zamknięcie liniowego ogranicznika LaTeX",
+	"LaTeX Block Open Delimiter": "Otwarcie blokowego ogranicznika LaTeX",
+	"LaTeX Block Close Delimiter": "Zamknięcie blokowego ogranicznika LaTeX",
+	"PlantUML Server": "Serwer PlantUML",
+	"Community": "Społecznośc",
+	"Subscribe to Newsletter": "Zapisz do Newsletter",
 	"GitHub": "GitHub",
 	"Blog": "Blog",
-	"Facebook Group": "Facebook Group",
+	"Facebook Group": "Grupa Facebook",
 	"Twitter": "Twitter",
-	"About": "About",
+	"About": "O nas",
 	"Boostnote": "Boostnote",
-	"An open source note-taking app made for programmers just like you.": "An open source note-taking app made for programmers just like you.",
-	"Website": "Website",
+	"An open source note-taking app made for programmers just like you.": "Społecznościowa aplikacja do przechowywania notatek, stworzona dla programistów takich jak Ty.",
```

Comment:
@mbarczak We have just updated this section to something like this: 'Aplikacja open-source do przechowywania notatek, stworzona dla programistów takich jak Ty". It's even better because it doesn't break the design